settings.py

```diff
@@ -270,6 +270,9 @@ def dump(self, f: TextIO, level: int = 0) -> None:
             # fetch class to avoid going through getattr
             cls = self.__class__
             type_hints = cls.get_type_hints()
+            if not type_hints:
+                # write empty dict for empty Group
+                cls._dump_value(type_hints, f, indent="  " * level)
```

Comment:
This adds `{}` if the **class** is empty, but loading a yaml can add values to the **instance** that will then get dumped in line 289.